src/SFA.DAS.FindEpao.Api/Models/GetCourseListItem.cs

```diff
@@ -16,7 +17,8 @@ public class GetCourseListItem
                 Id = standard.LarsCode,
                 Level = standard.Level,
                 Title = standard.Title,
-                IntegratedApprenticeship = standard.IntegratedApprenticeship
+                IntegratedApprenticeship = standard.IntegratedApprenticeship,
+                Versions = standard.Versions
```

Comment:
finnicky but the spec has it as StandardVersions, but I think Versions is fine.

Reply:
Changed for consistency to StandardVersions